cedar-lean/Cedar/Thm/Validation/Validator.lean

```diff
@@ -0,0 +1,236 @@
+import Cedar.Spec
+import Cedar.Data
+import Cedar.Validation
+import Cedar.Thm.Validation.Typechecker
+import Cedar.Thm.Data
+
+namespace Cedar.Thm
+
+open Cedar.Data
+open Cedar.Spec
+open Cedar.Validation
+
+-- for a single expression, evaluates to a boolean value (or appropriate error)
```

Comment:
Use [doc comments](https://leanprover-community.github.io/contribute/doc.html) where possible, so your notes get added to our neat [auto-generated docs page](https://cedar-policy.github.io/cedar-spec/docs/)

Reply:
I'll change the comments i have for now, but I'll add additional comments to meet the style guidelines in a future pass.